src/test/kotlin/three/consulting/epoc/controller/ProjectControllerIntegrationTest.kt

```diff
@@ -0,0 +1,77 @@
+package three.consulting.epoc.controller
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import three.consulting.epoc.utils.jsonPostEntity
+import java.net.URI
+
+class ProjectControllerIntegrationTest : ControllerIntegrationTest() {
+
+    @Test
+    fun `get project for id returns 200`() {
+        val response = restTemplate.getForEntity("/project/1", ObjectNode::class.java)
+        assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+    }
+
+    @Test
+    fun `add project returns 200`() {
+        val httpEntity = jsonPostEntity("src/test/resources/project/validCreation.json")
```

Comment:
Now that I think of it you could include the `src/test/resources/` part in the jsonPostEntity function itself already, as it repeats now everywhere and the function is only used in tests.

Reply:
Should I make the change here on do a separate PR for all the paths?

Reply:
Do as you wish. Could be a separate PR too to not bloat this one too much.